From 9cb9785188edcaf6ef241307db5ff4786c6c2d26 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 13 Mar 2023 23:15:16 +0000
Subject: [PATCH 09/17] gnu: cl-glsl-toolkit: Update to 1.0.0-2.4c4889e.

* gnu/packages/lisp-xyz.scm (sbcl-glsl-toolkit): Update to 1.0.0-2.4c4889e.
---
 gnu/packages/lisp-xyz.scm | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 53f5d4f698..19e20a26f9 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -19064,8 +19064,8 @@ (define-public cl-3d-transforms
   (sbcl-package->cl-source-package sbcl-3d-transforms))
 
 (define-public sbcl-glsl-toolkit
-  (let ((commit "d00ba1906e3b5eb08ea346ac300a1e77bb999d04")
-        (revision "1"))
+  (let ((commit "4c4889e75c635772c4df70b11d6f14e7a596da43")
+        (revision "2"))
     (package
       (name "sbcl-glsl-toolkit")
       (version (git-version "1.0.0" revision commit))
@@ -19077,7 +19077,7 @@ (define-public sbcl-glsl-toolkit
                (commit commit)))
          (file-name (git-file-name "glsl-toolkit" version))
          (sha256
-          (base32 "0as5796yazchq1qkna3srxlz5v7cf7ffny9cbqi41wsa2s20vbh9"))))
+          (base32 "0vrk8nqijqdi1rwf85rhd93yj7q8qkxnigxn3bw0cbrzvzrks9rl"))))
       (build-system asdf-build-system/sbcl)
       (inputs
        (list sbcl-cl-ppcre sbcl-documentation-utils sbcl-parse-float
-- 
2.39.2

